packages/block-editor/src/components/media-replace-flow/index.js

```diff
@@ -131,68 +124,68 @@ const MediaReplaceFlow = (
 	}
 
 	return (
-		<MediaUpload
-			onSelect={ ( media ) => selectMedia( media ) }
-			onClose={ () => setShowMediaReplaceOptions( true ) }
-			allowedTypes={ allowedTypes }
-			render={ ( { open } ) => (
-				<Toolbar className={ 'media-replace-flow components-dropdown-menu' }>
+		<Dropdown
```

Comment:
There is also this DropdownMenu component. Ou of curiosity, why isn’t it a good fit? We clearly need a ToolbarGroup which is collapsible and renders as button with menu. @diegohaz started some explorations.

Reply:
The reason it's not a DropdownMenu is because it contains elements that are not menu items (the URL input). There's some discussion about this in the original PR.

Reply:
Right, I'm personally wondering whether there should be any menu items used in this popover in that case. It might be a bit confusing for VoiceOver users.